src/components/ChatLog.jsx

```diff
@@ -0,0 +1,37 @@
+import './ChatLog.css';
+import PropTypes from 'prop-types';
+import ChatEntry from './ChatEntry';
+
+
+const ChatLog = ({entries, onLikeToggle}) => {
+    return (
+        <div className="chat-log">
+            {entries.map(entry => (
+                <ChatEntry
+                    key={entry.id}
+                    id={entry.id}
+                    sender={entry.sender}
+                    body={entry.body}
+                    timeStamp={entry.timeStamp}
+                    liked={entry.liked}
+                    onLikeToggle={onLikeToggle}
+                />
+            ))}
+        </div>
+    );
+};
```

Comment:
It's more typical to see the result of using `map` referenced by a variable and using that variable in the JSX, like:
```js
    const chatLogs =  entries.map((entry) => {
	      return (
	          <ChatEntry                                 
	            key={entry.id}
	            id={entry.id}
	            sender={entry.sender}
	            body={entry.body}
	            timeStamp={entry.timeStamp}
	            liked={entry.liked}
	            handleLike={handleLike}
	            currentUser={currentUser}
	          />
	        );
	     });
    };

  return (
        <section>
            <ul>{chatLogs}</ul>
        </section>
  );
```